adapters/InMobi/InMobiAdapter/GADMAdapterInMobiRewardedAd.m

```diff
@@ -143,9 +143,14 @@ - (void)requestRewardedAd {
 }
 
 - (void)presentFromViewController:(nonnull UIViewController *)viewController {
-  if ([_rewardedAd isReady]) {
-    [_rewardedAd showFromViewController:viewController];
-  }
+    if ([_rewardedAd isReady]) {
+        [_rewardedAd showFromViewController:viewController];
+    }  else {
+        NSError *error = GADMAdapterInMobiErrorWithCodeAndDescription(
+            GADMAdapterInMobiErrorAdNotReady,
+            @"[InMobi] Error - Rewarded ad not ready to be present.");
```

Comment:
Nit. can update to: `"InMobi SDK failed to present a rewarded ad."`

Reply:
done